support-frontend/assets/pages/[countryGroupId]/components/oneTimeCheckoutComponent.tsx

```diff
@@ -463,6 +463,9 @@ export function OneTimeCheckoutComponent({
 				});
 				const thankYouUrlSearchParams = new URLSearchParams();
 				thankYouUrlSearchParams.set('contribution', finalAmount.toString());
+				'paymentStatus' in paymentResult &&
```

Comment:
Why do we need to check this?

Reply:
paymentResult could be `PaymentResult` | `CreatePayPalPaymentResponse`. `userType` isn't on the payPal type (although possibly could be). So it's a way of type switching that we use further down too

Reply:
(Really sorry if I'm missing something obvious!) Should it not be `'userType' in paymentResult` in that case?

Reply:
Yep that makes more sense, it was a copy paste from below - my initial intent was to add it to PayPalResponse